packages/v4/src/content/design-guidelines/components/helper-text/helper-text.md

```diff
@@ -0,0 +1,46 @@
+---
+id: Helper text 
+section: components
+---
+
+Helper text is an on-screen field guideline that helps provide context regarding field inputs. 
+<br/>
+## Usage:
+**Helper text** is most commonly used in conjuction with form elements to provide additional hints and information for the user. It can also be used to explain why certain information is being requested, how to use a specific input or to suggest helpful hints. Helper text should always be visible and appears as a subheading, unlike [popover](https://www.patternfly.org/v4/components/popover/design-guidelines/) which display content in a new window.
+<br/>
+### When to use helper text:
+* Use helper text to provide imperative information regarding an input, such as character limits, password expectations, etc.  
```

Comment:
I just remembered we have a content rule about not using "etc" and instead using "and more" or something along those lines. (see here: https://patternfly-org-pr-3008-v4.surge.sh/v4/ux-writing/terminology#abbreviations-and-acronyms)